src/main/java/org/tdl/vireo/AppContextInitializedHandler.java

```diff
@@ -24,11 +31,37 @@
     private Boolean showBeans;
     
     @Autowired
+    private Environment env;
+    
+    @Autowired
     ApplicationContext applicationContext;
     
+    final static Logger logger = LoggerFactory.getLogger(AppContextInitializedHandler.class);
+    
     @Override
     protected void before(ContextRefreshedEvent event) {
 
+    	// Why not configure context here?
```

Comment:
I guess this code can go anywhere as long as it overrides internal property values and doesn't skip loading them instead. I did it as a static {} code in application.java just because it was easy and didn't require any annotations.